gneiss/balances.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import division
+import numpy as np
+from skbio.stats.composition import clr_inv
+from collections import OrderedDict
+
+
+def _balance_basis(tree_node):
```

Comment:
Yes, it seems like all these lines can be within _count_matrix but perhaps I'm missing something obvious. Could you clarify?